src/lib/core/selection/pseudo-checkbox/pseudo-checkbox.ts

```diff
@@ -5,9 +5,18 @@ import {
   ElementRef,
   Renderer2,
 } from '@angular/core';
+import {IsColorable, mixinColor} from '../../common-behaviors/color';
 
 export type MdPseudoCheckboxState = 'unchecked' | 'checked' | 'indeterminate';
 
+
+// Boilerplate for applying mixins to MdChip.
+export class MdPseudoCheckboxBase {
+  constructor(public _renderer: Renderer2, public _elementRef: ElementRef) {}
```

Comment:
Should do this with properties rather than a constructor because [the constructor approach will generate more code](http://www.typescriptlang.org/play/#src=class%20R%20%7B%20%7D%0Aclass%20E%20%7B%20%7D%0A%0A%2F%2F%20----------------------------------------------------------------------------------------%0A%0Aexport%20class%20WithConstructor%20%7B%0A%20%20constructor(public%20_renderer%3A%20R%2C%20public%20_elementRef%3A%20E)%20%7B%7D%0A%7D%0A%0A%2F%2F%20----------------------------------------------------------------------------------------%0A%0Aexport%20class%20WithProperties%20%7B%0A%20%20_renderer%3A%20R%3B%0A%20%20_elementRef%3A%20E%3B%0A%7D%0A%0A%2F%2F%20----------------------------------------------------------------------------------------%0A%0A%0Aclass%20End%20%7B%20%7D).